xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -22,7 +22,7 @@ define('xwiki-lightbox-messages', {
   keys: [
     'date',
     'author',
-    'toggle.label'
+    'toggler.label'
```

Comment:
Where are you using this key? As far as I can see, this key isn't used in JavaScript and should be removed.

Reply:
I just remembered to check that this key is actually useful after sending the message, way too late ^^'
Sorry, this is addressed in https://github.com/xwiki/xwiki-platform/pull/2330/commits/b25c4bf8b3be2cf2e79a1a58820946ad229df6fa.